time_tracker_api/time_entries/time_entries_repository.py

```diff
@@ -93,51 +73,34 @@ def count(
         self,
         event_context: EventContext,
         conditions: dict = None,
-        custom_sql_conditions: List[str] = None,
+        owner_ids: list = None,
         date_range: dict = None,
         visible_only=True,
         **kwargs,
     ):
         conditions = conditions if conditions else {}
-        custom_sql_conditions = (
-            custom_sql_conditions if custom_sql_conditions else []
-        )
         date_range = date_range if date_range else {}
 
-        custom_sql_conditions.append(
-            self.create_sql_date_range_filter(date_range)
-        )
+        date_range_params = self.generate_params(date_range)
+        params = self.generate_params(conditions)
+        params.extend(date_range_params)
 
-        custom_params = self.generate_params(date_range)
-        partition_key_value = self.find_partition_key_value(event_context)
-        params = [
-            {"name": "@partition_key_value", "value": partition_key_value},
-        ]
-        params.extend(self.generate_params(conditions))
-        params.extend(custom_params)
-
-        query_str = """
-            SELECT VALUE COUNT(1) FROM c
-            WHERE c.{partition_key_attribute}=@partition_key_value
-            {conditions_clause}
-            {visibility_condition}
-            {custom_sql_conditions_clause}
-            """.format(
-            partition_key_attribute=self.partition_key_attribute,
-            visibility_condition=self.create_sql_condition_for_visibility(
-                visible_only
-            ),
-            conditions_clause=self.create_sql_where_conditions(conditions),
-            custom_sql_conditions_clause=self.create_custom_sql_conditions(
-                custom_sql_conditions
-            ),
+        query_builder = (
+            CosmosDBQueryBuilder()
+            .add_select_conditions(["VALUE COUNT(1)"])
```

Comment:
If you consider feasible, consider adding a count method to the general query builder.

Reply:
My recommendation is to keep using the select_conditions method because it allows any expression to use in a centralized and flexible way.

    def build(self):
        self.query = """
        SELECT {select_conditions} FROM c
        {where_conditions}